components/grid/selection/overview.md

```diff
@@ -10,196 +10,72 @@ position: 0
-            });
-        }
-    }
+When you select a row or a cell, they will be highlighted to notify you that they are selected. By default you can click on a row or on a cell to select it. You can also select a row through a checkbox column. To select multiple rows or cells, use the `Ctrl` or `Shift` key to extend the selection.
 
-    public class Employee
-    {
-        public int EmployeeId { get; set; }
-        public string Name { get; set; }
-        public string Team { get; set; }
-    }
-}
-````
+See [Rows Selection Options]({%slug components/grid/selection/rows%}#rows-selection-options) and [Cells Selection Options]({%slug components/grid/selection/cells%}#cells-selection-options) for more details.
 
+## Selected Items
 
-## Notes
+You can get and set the selected rows or cells:
+* To access the selected rows add the Grid `SelectedItems` parameter.
+* To access the selected cells add the Grid `SelectedCells` parameter.
 
-### Editing Modes
+Both parameters are collections. You can use the parameters to manipulate the selected rows or cells. You can also use the parameters to pre-select a row or cell for your users.
 
-#### InCell Edit Mode
+See [Selected Rows]({%slug components/grid/selection/rows%}#selected-rows) and [Selected Cells]({%slug components/grid/selection/cells%}#selected-cells) for more details.
 
-In the [Incell EditMode]({%slug components/grid/editing/incell%}) selection can be applied only via a [checkbox column]({%slug components/grid/columns/checkbox%}) (`<GridCheckboxColumn />`). This is required due to the overlapping action that triggers selection and InCell editing (clicking in the row) - if row click selection was enabled with InCell editing, each attempt to select a row would put a cell in edit mode; and each attempt to edit a cell would select a new row. Such user experience is confusing, and so selection will only work through the row selection checkbox.
+## Selection Events
 
-To see how to select the row that is being edited in InCell edit mode without using a `<GridCheckboxColumn />` check out the [Row Selection in Edit with InCell EditMode]({%slug grid-kb-row-select-incell-edit%}) Knowledge Base article.
+You can respond to the user action of selecting a new item through the Grid events:
+* Use the [`SelectedItemsChanged` event]({%slug components/grid/selection/rows%}#selecteditemschanged) to respond to row selection.
+* Use the [`SelectedCellsChanged` event]({%slug components/grid/selection/cells%}#selectedcellschanged) to respond to cell selection.
 
-#### Inline and Popup Edit Modes
+> Both binding to the property and using its event cannot be used at the same time as Blazor allows only one. To use the `SelectedItemsChanged` event, utilize one-way binding for the `SelectedItems` property. Otherwise, use two-way binding for the `SelectedItems` property without the `SelectedItemsChanged` event. The same applies to the `SelectedCells` and the `SelectedCellsChanged`.
```

Comment:
This is simple general knowledge and we should include it only if we are sure that customers will stumble upon this.

If you decide to leave the note, it needs simplification and refactoring.